docs/build/unreleased/695.rst

```diff
@@ -0,0 +1,7 @@
+.. change::
+    :tags: change
```

Comment:
let's classify this as:

    :tags: bug, commands

Reply:
Updated fd3e3b8.

Reply:
thanks1